src/main/java/SubwayLines.java

```diff
@@ -23,8 +23,8 @@
     List<Integer> P;
     List<Integer> Q;
     List<Integer> R;
-    List<Integer> S;
-    List<Integer> T;
+//    List<Integer> S;
+//    List<Integer> T;
```

Comment:
Why did you take out the T train?

Reply:
It was taken out of the new Json that you sent